### PR TITLE
Dashboard tabs visible to course team

### PR DESCRIPTION
**Pull request by tasawernawaz**
[ECOM-7418](https://openedx.atlassian.net/browse/ECOM-7418)

* Course team can see **IN DEVELOPMENT**, **PREVIEW READY** and **PUBLISHED COURSE RUNS**.
* Only users added in **PROJECT COORDINATOR** group will see **STUDIO REQUEST** tab.

@awais786 @waheedahmed @bderusha Please review.

### CODE REVIEW

**Review by awais786 — APPROVED** [review]

File: course_discovery/templates/publisher/dashboard.html [hunk 1]
```diff
@@ -13,22 +13,23 @@ <h2 class="hd-2 emphasized">{% trans "Course runs" %}</h2>
...
-            {% if can_view_all_tabs %}
-                <li role="tab" id="tab-preview" class="tab" aria-selected="false" aria-expanded="false"
-                    aria-controls="preview" tabindex="-1">
-                    <span>{{ preview_count }}</span>{% trans "PREVIEW READY" %}
-                </li>
+            <li role="tab" id="tab-preview" class="tab" aria-selected="false" aria-expanded="false"
+                aria-controls="preview" tabindex="-1">
+                <span>{{ preview_count }}</span>{% trans "PREVIEW READY" %}
+            </li>
+            {% if is_project_coordinator %}
```
Comment (waheedahmed) [inline course_discovery/templates/publisher/dashboard.html:22]: Add this check on its template also.

**Review by waheedahmed — APPROVED** [review]
Only one comment, 👍 once addressed.

File: course_discovery/templates/publisher/dashboard.html (hunk 1 above)
Comment (tasawernawaz) [reply course_discovery/templates/publisher/dashboard.html:22]: added.

Comment (waheedahmed) [reply course_discovery/templates/publisher/dashboard.html:22]: :shipit:

**Pull request merged**